modules/qualitycontrol/ReceiveVideoController.js

```diff
@@ -7,6 +7,25 @@ import { MediaType } from '../../service/RTC/MediaType';
 const logger = getLogger(__filename);
 const MAX_HEIGHT = 2160;
 const LASTN_UNLIMITED = -1;
+const ASSUMED_BANDWIDTH_BPS = {
+    MAX_VALUE: 10000000,
```

Comment:
Is this the max? I thought it's now capped by the JVB setting, so then we should probably not limit it here?

Reply:
Yes, it should be the max we allow, I'll remove it from here.